packages/components/src/internal/renderers/StorageStatusRenderer.tsx

```diff
@@ -14,7 +14,11 @@ export class StorageStatusRenderer extends React.PureComponent<StorageStatusProp
         if (value?.toLowerCase() === 'not in storage' || value?.toLowerCase() === 'discarded') {
```

Comment:
I'm seeing an issue where `data` is undefined

```suggestion
        const { data } = this.props;
        if (!data) return null;
        
        const value = data.get('value', null);
        if (value?.toLowerCase() === 'not in storage' || value?.toLowerCase() === 'discarded') {
```